source/admin_manual/list_of_events.rst

```diff
@@ -24,6 +24,10 @@ Root Categories
 +--------------------+---------------------------------------------------------+
 | auth-client        | Authentication client library events.                   |
 +--------------------+---------------------------------------------------------+
+| dict               | Dictionary related events                               |
++--------------------+---------------------------------------------------------+
+| dict-server        | Dictionary server events                                |
```

Comment:
These could be clarified a bit maybe.. Also I tried to remove "events" word from these categories to avoid constant repetition (although for some reason they are still in auth & auth-client, should remove from there too). Maybe:

 * dict : Dictionary library (also used by dict process)
 * dict-server : Dictionary server/proxy (dict process)